.github/actions/gen-metadata-file/action.yml

```diff
@@ -0,0 +1,71 @@
+name: generate metadata info file for the run
+description: 'generate metadata file'
+inputs:
+  reportportal_host:
+    description: "reportportal hostname"
+    required: true
+  project:
+    description: "reportportal project"
+    required: true
+  gha_url:
+    description: "GHA job url"
+    required: true
+  wf_category:
+    description: "workflow category; will be used in the droute attributes"
+    required: true
+  gitref:
+    description: "gitref for checked out product code; will be used in the droute attributes"
+    required: true
+  run_name:
+    description: "run name; if empty, will use project as default"
+  additional_attributes_string:
+    description: "additional information for droute attributes, specified in the 'key=value, key=value, ...' format"
+outputs:
+  metadata_filepath:
+    description: "path of the metadata file"
+    value: ${{ steps.metadata.outputs.metadata_filepath }}
+runs:
+  using: composite
+  steps:
+
+    - name: generate metadata file
+      id: metadata
+      run: |
+
+        # wheel is downloaded already
+        WHEEL=`find . -type f -name compressed*.whl -exec basename {} \;`
+
+        RUN_NAME=${{ inputs.run_name }}
+        if [ -z "${{ inputs.run_name }}" ]; then
+          RUN_NAME=${{ inputs.project }}
+        fi
+        JSON_STRING=$(cat <<EOF
+        {
+          "hostname": "$REPORTPORTAL_HOST",
+          "project": "${{ inputs.project }}",
+          "description": "GHA job url: ${{ inputs.gha_url }}",
+          "name": "${RUN_NAME}",
+        EOF
```

Comment:
missing a closing `}`

Reply:
This is intended, } is added later once other fields are all added.